source/09-audit/01-the-audit-process.rst

```diff
@@ -5,14 +5,36 @@
-* Upon completion of the audit, the auditor is REQUIRED to send a signed copy of the audit report to the COUNTER office (compliance@counterusage.org). On receipt of the successful audit report, the report provider will be sent a dated COUNTER logo, which they can display on their website. For example:
 
-  .. figure:: ../_static/img/91-counter-audit-logo.jpg
-     :alt: COUNTER audit logo
-     :width: 30%
+Audit Scope
+"""""""""""
 
-* The dated logo MUST link to the report provider’s entry on the COUNTER Registry.
-* Failure to complete a successful audit by the due date may result in COUNTER removing that report provider from the list of compliant report providers on the COUNTER Registry.
+Audits are primarily focused on the four COUNTER Reports. The Standard Views of the COUNTER Reports will pass the audit provided the metrics match those in the COUNTER Report, with the appropriate aggregation.
+
+Each report provider MUST be audited on all the COUNTER Reports and Standard Views of the COUNTER Reports they are required to provide. Report providers who elect to offer additional COUNTER Reports (e.g. eJournal Host_Types that offer Item Reports) SHOULD have the additional COUNTER Reports and Standard Views of the COUNTER Reports audited. If the elective reports are not audited, this must be documented in the audit report.
+
+Any exclusions to the audit scope MUST be agreed with the Project Director in writing prior to an audit commencing.
+
+
+Audit Scope For Third-Party Report Providers
+""""""""""""""""""""""""""""""""""""""""""""
+
+Some publishers work with third parties to provide COUNTER reports. In these cases it is the third party report provider that is audited. Where such third parties are supporting multiple platforms, they must provide the auditor with a list of all platforms and the related COUNTER Reports and Standard Views. The auditor will then select a minimum of two and a maximum of five platforms from the list and audit accordingly, with the list to be approved by the COUNTER Project Director before the audit begins.
+
+Where different technologies are used to generate reports for different platforms (e.g. data is harvested differently from one platform to the next), each technology MUST be audited separately.
+
+
+Stages of the Audit Process
+"""""""""""""""""""""""""""
+
+Audits follow a five-stage process.
+
+* **Stage one** - pre-flight preparation. Report providers SHOULD run all of their COUNTER reports through the COUNTER Validation Tool. Report providers SHOULD also seek feedback from one of our listed library groups in respect of issues they have identified with the reports. Report providers SHOULD fix any problems that are discovered and re-run the Validation Tool before proceeding to Stage two.
```

Comment:
COUNTER Reports and Standard Views [of COUNTER Reports]?

A link or some other kind of reference is needed for "our listed library groups".